Services/UserService.cs

```diff
@@ -0,0 +1,30 @@
+using System.Linq;
+using System.Threading.Tasks;
+using AutoMapper;
+using PictureApp.API.Data;
+using PictureApp.API.Data.Repository;
+using PictureApp.API.Models;
+using PictureApp_API.Dtos;
+
+namespace PictureApp_API.Services
+{
+    public class UserService : IUserService
+    {
+        private readonly IRepository<User> userRepo;
+        private readonly IUnitOfWork unitOfWork;
+        private readonly IMapper mapper;
+        public UserService(IRepository<User> userRepo, IUnitOfWork unitOfWork, IMapper mapper)
```

Comment:
I would like to encourage you to check whether dependencies are null or not :)

Reply:
resolve by: https://github.com/anowakowski/PictureApp_API/pull/17/commits/a5e9b125df36d84fc4586c585d9f5f7a4b681585